install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```diff
@@ -124,6 +132,7 @@ spec:
         - --cluster-id=$(CLUSTER_ID)
         - --kvstore-opt
         - etcd.config=/var/lib/cilium/etcd-config.yaml
+        - --clustermesh-health-port=8080
```

Comment:
How is the health port used? Are we able to just change it in this one place and all will work? I would have expected that we (a) may need to also change the client configuration to access this port (or at least container configuration to map a system port to this port), and (b) may even need to consider how this could impact upgrade as existing users upgrade to this version of the Helm chart.

Reply:
Honestly, I did not find any usage of this health port. There is no `ports` definition for the api server and also the service is only defining the etcd port `2379`.

So I just *tried* changing it and find no problems.

My knowledge about the api server ist quite limited, but it is probably a good idea to use this health port for proper liveness/readiness probes. But this is out of scope for this PR IMHO.

Reply:
Hmm, okay. @cilium/sig-clustermesh could you comment on this?

Reply:
I heard back from @YutaroHayakawa that this never had a readiness probe or anything attached to it, so it should be fine to change it like this. I've filed https://github.com/cilium/cilium/issues/23893 to follow up on adding probes.